### PR TITLE
added sound

### PR DESCRIPTION
**Pull request by lucien386**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ before_install:
 - pip install -U pytest pytest-cov
 - pip install codecov
 - sudo apt-get install lilypond
+- sudo apt-get install musescore
+- sudo apt-get install xvfb
+- Xvfb :0 -screen 0 1280x768x24&
```
Comment (xicheng87) [inline .travis.yml:13]: Mind clarifying in comment what 1280x768x24& is referring to?

File: tests/test_image.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from music21 import *
 
 def test_png():
-    a = converter.parse('tests/sample.musicxml').write('lilypond.png', 'tests/result')
+    a = converter.parse('tests/sample.musicxml').write('lilypond.png')
+    b = converter.parse('tests/sample.musicxml').write('musicxml.png')
     assert 1 == 1
```
Comment (xicheng87) [inline tests/test_image.py:6]: This line is meaningless, delete. 

Furthermore, I think you need to add a few more test cases for musicxml, we need to test something up to size of several MBs, to ensure that things work.

File: tests/test_sound.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from music21 import *
+
+def test_midi():
+    a = converter.parse('result.abc').write('midi')
+    assert 1 == 1
```
Comment (xicheng87) [inline tests/test_sound.py:5]: Remove this line as it does nothing.

File: .travis.yml (hunk 1 above)
Comment (lucien386) [reply .travis.yml:13]: it does not quite matter; xvfb is used to simulate a window opened for musescore.

Comment (lucien386) [reply .travis.yml:13]: 1280x768x24& is the param for that fake window

**Pull request merged**